pages/school-administrators.md

```diff
@@ -90,23 +81,13 @@ relatedlinks:
 
 <div>
   <p class="va-introtext">
-    Trainings, resources, guides, and information on GI Bill® programs geared towards School Certifying Officials (SCOs) and school administrators.
+    Trainings, resources, guides, and information on GI Bill® programs created for school administrators and certifying officials.
   <p>
 </div>
 
-### On this page
-
-<div>
-  <ul>
-    <li><a href="#handbooks" onClick="recordEvent({ event: 'nav-jumplink-click'});">Handbooks</a></li>
-    <li><a href="#trainings-and-webinars" onClick="recordEvent({ event: 'nav-jumplink-click'});">Trainings and Webinars</a></li>
-    <li><a href="#program-approvals" onClick="recordEvent({ event: 'nav-jumplink-click'});">Program approvals information</a></li>
-    <li><a href="#upload-files" onClick="recordEvent({ event: 'nav-jumplink-click'});">Upload files to VA</a></li>
-    <li><a href="#other-resources" onClick="recordEvent({ event: 'nav-jumplink-click'});">Other resources for schools</a></li>
-  </ul>
-</div>
+<div><va-on-this-page class="vads-u-margin-left--1 vads-u-margin-bottom--0 vads-u-padding-bottom--0" /></div>
 
-<section class="usa-grid">
+<section>
   <div class="va-h-ruled--stars"></div>
```

Comment:
Removing this class allows this separator to extend to the full width of the column, as in the design.